app/views/pair_requests/_pair_request.html.erb

```diff
@@ -5,19 +5,17 @@
   </p>
 
   <p>
-    <strong>Acceptor:</strong>
-    <% if pair_request.acceptor.present? %>
-      <% if pair_request.acceptor == current_user %>
-        <%= link_to "Unaccept", pair_request_acceptance_path(@pair_request), data: { turbo_method: :delete }, class: "btn btn-danger" %>
-      <% else %>
-        <%= pair_request.acceptor.email %>
-      <% end %>
-    <% else %>
-      <% if pair_request.author == current_user %>
-        Not yet accepted
-      <% else %>
-        <%= link_to "Accept this request", pair_request_acceptances_path(pair_request), data: { turbo_method: :post }, class: "btn btn-success" %>
-      <% end %>
+    <strong>Invitee: <%= pair_request.invitee.email %></strong>
+  </p>
+
+  <p>
+    <strong>Status: <%= pair_request.status %></strong>
+    <% if pair_request.invitee == current_user && pair_request.pending? %>
```

Comment:
This is fine for now. When I integrate pundit in we will go back and fix these up.

Reply:
should be able to handle using pundit now.